File: src/main/java/my/maryvk/maryvkweb/service/RelationChangeService.java
package my.maryvk.maryvkweb.service;

import my.maryvk.maryvkweb.domain.RelationChange;

import java.util.List;

public interface RelationChangeService {
    void registerChange(RelationChange relationChange);

    List<RelationChange> findAllByOwnerIdOrderByTimeDesc(int ownerId);
    List<RelationChange> findAllOrderByTimeDesc();
}

File: src/main/java/my/maryvk/maryvkweb/service/RelationChangeService.kt
package my.maryvk.maryvkweb.service

import my.maryvk.maryvkweb.domain.RelationChange

interface RelationChangeService {
    fun registerChange(relationChange: RelationChange)

    fun findAllByOwnerIdOrderByTimeDesc(ownerId: Int): List<RelationChange>
    fun findAllOrderByTimeDesc(): List<RelationChange>
}